main.sh

```diff
@@ -29,18 +30,31 @@ if [[ "${GITHUB_REF:?}" != "refs/tags/"* ]]; then
 fi
 tag="${GITHUB_REF#refs/tags/}"
 
-# TODO: Support custom prefix of tags https://github.com/taiki-e/create-gh-release-action/issues/1
-if [[ ! "${tag}" =~ ^v?[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z_0-9\.-]+)?(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
+if [[ ! "${tag}" =~ ^${prefix}-?v?[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z_0-9\.-]+)?(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
     error "invalid tag format: '${tag}'"
     exit 1
 fi
-if [[ "${tag}" =~ ^v?[0-9\.]+-[a-zA-Z_0-9\.-]+(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
+if [[ "${tag}" =~ ^${prefix}-?v?[0-9\.]+-[a-zA-Z_0-9\.-]+(\+[a-zA-Z_0-9\.-]+)?$ ]]; then
     prerelease="--prerelease"
 fi
-version="${tag#v}"
+
+version="${tag}"
+# extract the portion of the tag matching the prefix pattern
+if [[ ! "${prefix}" = "" ]]; then
+    prefix=$(grep <<<"${tag}" -Eo "^${prefix}")
+    version="${tag#${prefix}}"
```

Comment:
```suggestion
    version="${tag#"${prefix}"}"
```
https://github.com/taiki-e/create-gh-release-action/runs/4590088424?check_suite_focus=true
```
In main.sh line 45:
    version="${tag#${prefix}}"
                   ^-------^ SC2295 (info): Expansions inside ${..} need to be quoted separately, otherwise they match as patterns.

Did you mean: 
    version="${tag#"${prefix}"}"

For more information:
  https://www.shellcheck.net/wiki/SC2295 -- Expansions inside ${..} need to b...
```